src/Kmd.Logic.Gateway.Automation/Gateway/Api.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Logic.Gateway.Automation.Gateway
+{
+    public class Api
+    {
+        public string Name { get; set; }
+
+#pragma warning disable CA2227 // Collection properties should be read only
```

Comment:
Please fix this issue - either IReadOnlyList or IList with only getter

Reply:
IReadOnlyList or IList with only getter will not work it will throw exception while deserialization. I have considered IEnumerable